client/views/answers/answers.js

```diff
@@ -35,12 +35,12 @@ Template.answers.helpers({
     answers.reverse();
     for (let a = 0; a < answers.length; a++) {
       answers[a].text = answers[a].text.replace(/\B(@\S+)/g, '<strong>$1</strong>');
+      // eslint-disable-next-line max-len
```

Comment:
Okay so I've replaced this regex with the SimpleSchema URL regex locally and it works fine for valid URLs, but doesn't cover URLs without a protocol specified (for example, before this change google.com would be highlighted and links to http://google.com, but now because "google.com" is an invalid URL according to the schema, this change doesn't happen). Do I commit and push this change anyway?